tests/integration/run_tests.py

```diff
@@ -176,7 +176,7 @@ def run_shell_command(command: str, command1: str, expected_response: str, verbo
     return 0
 
 def run_tests(test_dir: str, output_dir: str, json_file: str, verbose: bool, daemon_under_test: str, exit_on_fail: bool, verify_with_daemon: bool, daemon_as_reference: str,
-              dump_output: bool, test_number, infura_url: str):
+              dump_output: bool, test_number, infura_url: str, daemon_url: str):
```

Comment:
`daemon_url` -> `daemon_host` according to the meaning of `host` parameter in `get_target` function

Reply:
ok renamed deamon_on_host